packages/compass-global-writes/src/components/states/shard-key-correct.tsx

```diff
@@ -0,0 +1,210 @@
+          {customShardKeyField} field at insert time.
+        </strong>
+        {nbsp}We have included a table for reference below.
+      </Banner>
+
+      <div className={codeBlockContainerStyles}>
+        <Body data-testid="shardkey-description-title">
+          <strong>{namespace}</strong> is configured with the following shard
+          key:
+        </Body>
+        <Code language="js" data-testid="shardkey-description-content">
+          {shardKey.fields.map((field) => `"${field.name}"`).join(', ')}
+        </Code>
+      </div>
+
+      <Subtitle>Example commands</Subtitle>
+      <div className={paragraphStyles}>
+        <Body>
+          Start querying your database with some of the most{' '}
+          <Link
+            href="https://www.mongodb.com/docs/atlas/global-clusters"
+            hideExternalIcon
+          >
+            common commands
+          </Link>{' '}
+          for Global Writes.
+        </Body>
+        <Body>
+          Replace the text to perform operations on different documents. US-NY
+          is an ISO 3166 location code referring to New York, United States. You
+          can look up other ISO 3166 location codes below.
+        </Body>
+      </div>
+
+      <div className={codeBlockContainerStyles}>
+        <Label htmlFor="finding-documents">Finding documents</Label>
+        <Code
+          language="js"
+          data-testid="sample-finding-documents"
+          id="finding-documents"
+        >
+          {sampleCodes.findingDocuments}
+        </Code>
+      </div>
+
+      <div className={codeBlockContainerStyles}>
+        <Label htmlFor="inserting-documents">Inserting documents</Label>
+        <Code
+          language="js"
+          data-testid="sample-inserting-documents"
+          id="inserting-documents"
+        >
+          {sampleCodes.insertingDocuments}
+        </Code>
+      </div>
+
+      <Subtitle>Location Codes</Subtitle>
+      <div className={paragraphStyles}>
+        <Body>
+          Each document’s first field should include an ISO 3166-1 Alpha-2 code
+          for the location it belongs to.
+        </Body>
+        <Body>
+          We also support ISO 3166-2 subdivision codes for countries containing
+          a cloud provider data center (both ISO 3166-1 and ISO 3166-2 codes may
+          be used for these countries). All valid country codes and the zones to
+          which they map are listed in the table below. Additionally, you can
+          view a list of all location codes{' '}
+          <Link href="/static/atlas/country_iso_codes.txt">here</Link>.
+        </Body>
+        <Body>
+          {atlasMetadata?.projectId && atlasMetadata?.clusterName && (
+            <>
+              Locations’ zone mapping can be changed by navigating to this
+              clusters{' '}
+              <Link
+                href={`/v2/${atlasMetadata?.projectId}/clusters/edit/${atlasMetadata?.clusterName}`}
```

Comment:
Easy to miss, mms client uses hash router inside the project, so you need a hash here (which makes me think that we might want to put all the url logic like that in one place maybe so that it's easier to stay consistent, not an action for now, just a thought)

```suggestion
                href={`/v2/${atlasMetadata?.projectId}#/clusters/edit/${atlasMetadata?.clusterName}`}
```